recipes/pandasgui/meta.yaml

```diff
@@ -12,14 +12,22 @@ source:
 
 build:
   number: 0
-  noarch: python
+  skip: true  # [py<35]
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
+  build:
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxext') }}     # [linux]
```

Comment:
You can still be noarch and you probably don't need all of them. Please try adding only the yum requirements with `mesa-libGL` first.

Reply:
Ok, so I can safely remove the selectors in the complete recipe to make it noarch?

Reply:
Let's try first without the CDTs, if we can fix this with only the yum requirements we can make Linux GUI happy and macOS and Windows can probably use the same noarch package. If we really need the CDTs we may need to drop the noarch.

Reply:
Ok, I did not convert it back to noarch, yet, but I removed the CDT packages you mentioned in https://github.com/conda-forge/staged-recipes/pull/13237#discussion_r524367625 and the libGL error does not show up again. So, they are obviously not needed.

However, there is still the `ImportError: libXcomposite.so.1`. According to [this](https://github.com/spyder-ide/spyder/issues/6990#issuecomment-418960594), a `sudo apt-get install python-pyqt5.qtwebkit` might help. But where do I have to add it?

Reply:
Adding python-pyqt5.qtwebkit to the host requirements to fix the `ImportError: libXcomposite.so.1` does not seem to work. Do I need to use a different package name?